cmd/vicadmin/logs.go

```diff
@@ -255,10 +255,14 @@ func tarEntries(readers map[string]entryReader, out io.Writer) error {
 			log.Warningf("error reading %s(%s): %s\n", name, r, err)
 			continue
 		}
+		var sz int64
```

Comment:
What is the behavior when `e` is `nil`? `sz` gets set to `nil` below also? Is that going to cause a nil pointer exception somewhere else?